src/Apps/Order2/Routes/Details/Components/Order2DetailsPaymentInfo.tsx

```diff
@@ -26,7 +28,7 @@ export const Order2DetailsPaymentInfo: React.FC<Props> = ({ order }) => {
 
   return (
     <Box p={2} backgroundColor="mono0">
-      <Text variant="sm" fontWeight={500} color="mono100">
+      <Text variant="sm" fontWeight={700} color="mono100">
```

Comment:
I don't (think) we ever explicitly set the fontWeight like this; rather its just `fontWeight='bold'`

Reply:
Yea - we were looking at it and bold might work as well but it's 800 weight. 700 looks better in this context. Not a strong opinion though.... It works either way.